utils/utils.tsx

```diff
@@ -1178,12 +1191,12 @@ export function sortUsersByStatusAndDisplayName(users, statusesByUserId, teammat
 /**
  * Gets the entire name, including username, full name, and nickname, of the specified user
  */
-export function displayEntireNameForUser(user) {
+export function displayEntireNameForUser(user: UserProfile): string | JSX.Element {
```

Comment:
Would it make sense to use ReactElement?

```suggestion
export function displayEntireNameForUser(user: UserProfile): React.ReactElement {
```

Reply:
:+1:, opted for `React.ReactNode` since it needs to support strings, too.